source/georef/street_network.h

```diff
@@ -225,7 +228,9 @@ struct StreetNetwork {
 };
 
 /// Build a path from a reverse path list
-Path create_path(const GeoRef& georef, std::vector<vertex_t> reverse_path, bool add_one_elt);
+Path create_path(const GeoRef& georef,
+                 std::vector<vertex_t> reverse_path,
```

Comment:
Minor: Do we really want to pass a copy of a `vector<vertex_t>` to the function? maybe, i missed something...